client/packages/system/src/Name/Components/InternalSupplierSearchModal/InternalSupplierSearchModal.tsx

```diff
@@ -9,29 +11,32 @@ import { useName, NameRowFragment } from '../../api';
 import { filterByNameAndCode, NameSearchProps } from '../../utils';
 import { getNameOptionRenderer } from '../NameOptionRenderer';
 
-const InternalSupplierSearchComponent: FC<NameSearchProps> = ({
-  open,
-  onClose,
-  onChange,
-}) => {
+const InternalSupplierSearchComponent: FC<NameSearchProps> = props => {
```

Comment:
I don't see why the change to use `props` and then destructure later. Is there a reason? 
If not - can you revert to previous to be consistent.

Reply:
since the type is a union now, `open` and `onClose` only exist in a variant, so typescript complains if we destructure them without typeguard